plugins/pagerduty/plugin.spec.yaml

```diff
@@ -17,48 +19,63 @@ tags:
 - incident management
 hub_tags:
   use_cases: [alerting_and_notifications]
-  keywords: [incident response, incident]
+  keywords: [incident_response, incidents]
   features: []
+sdk: 
+  type: full 
+  version: 5 
+  user: nobody
 connection:
   api_key:
     title: API Key
     type: credential_secret_key
     description: API Key
     required: true
+    example: stRbCzL92kpAfwCkSiA9
 types:
   user:
     id:
       description: ID
       type: string
+      example: PYGDZB8
     self:
       description: URL to view object
       type: string
+      example: "https://api.pagerduty.com/users/PYGDZB8"
```

Comment:
Is this sensitive info?

Reply:
All of these examples were fake data or else taken from the pager duty api docs, but i'll update them to make them easier to look fake